settings.py

```diff
@@ -21,7 +21,6 @@ def path(*args):
 
 
 DEBUG = config('DEBUG', default=False, cast=bool)
-TEMPLATE_DEBUG = DEBUG
```

Comment:
The docs said that you can set it in `TEMPLATES` if you want it to be different than `DEBUG`. But we don't, so I thought just removing it would do what we want. Is that not the case?